tests/unit/src/formatMarkup.test.js

```diff
@@ -569,5 +569,18 @@ describe('Format markup', () => {
           </pre>
         `);
     });
+
+    test.only('WhiteSpace Preserved for PRE child elements', async () => {
+      const markup = [
+        '<pre>',
+        '  Hello World <div>Hello World <p>Hello World</p></div>',
+        '</pre>'
+      ].join('\n');
+
+      expect(markup)
+        .toMatchInlineSnapshot(`
+          <pre>  Hello World <div>Hello World <p>Hello World</p></div></pre>
```

Comment:
Shouldn't this be:
```suggestion
          <pre>
            Hello World <div>Hello World <p>Hello World</p></div>
          </pre>
```

Reply:
I expected the <pre> tag to maintain its original formatting, including line breaks. I thought it would be preserved as-is, without any additional spacing applied to the tag itself.

Reply:
The original input is

```js
'<pre>\n  Hello World <div>Hello World <p>Hello World</p></div>\n</pre>'
```

So if it was retain the original line breaks, the output would be the same as that input. Which it isn't. It is missing both `\n`'s.

Weirdly though, when testing in the browser, the first `\n` get removed:

* https://jsfiddle.net/whofztas

`parse5`'s generated AST also follows that same interpretation, ignoring the first `\n`, but including the rest.

So the expect should look like this:

```suggestion
          <pre>  Hello World <div>Hello World <p>Hello World</p></div>
          </pre>
```